src/main.rs

```diff
@@ -28,7 +29,10 @@ async fn main() -> Result<()> {
 
     HttpServer::new(move || {
         App::new().route("/api", web::get().to(index)).service(
-            web::scope("/api").app_data(web::Data::new(CompanyRepository::new(arc_pool.clone()))),
+            web::scope("/api")
+                // add handlers here
```

Comment:
This will be done through a separate configuration function due to making testing easier.
It's in the api-skeleton branch.